test/functional/repository/find-options-operators/repository-find-operators.ts

```diff
@@ -605,6 +605,16 @@ describe("repository > find options > operators", () => {
             { id: 3, likes: 3, title: "About #3" },
             { id: 6, likes: 6, title: "About #6" },
         ]);
+
+        // check operator
+        const result6 = await connection.getRepository(Post).find({
+            likes: Raw((columnAlias) => `${columnAlias} IN (:...values)`, { value: [2, 3, 6] }),
```

Comment:
You have typo in parameters. Should be { values: [2, 3, 6] }

Reply:
I did fix it and edit the previous commit (git fixup).

Now the tests are green ✅